scripts/update-frontend-canister.Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG RUST_VERSION
+FROM rust:${RUST_VERSION} AS builder
+RUN cargo install ic-wasm --version 0.2.0
```

Comment:
Please remove the corresponding `cargo install ic-wasm` from the github workflow